examples/addons/aad-pod-identity/README.md

```diff
@@ -0,0 +1,67 @@
+# AAD Pod Identity Add-on
+
+
+This is the AAD Pod Identity add-on.  Add this add-on to your json file as shown below to automatically enable AAD Pod identity in your new Kubernetes cluster.
+
+```json
+{
+    "apiVersion": "vlabs",
+    "properties": {
+      "orchestratorProfile": {
+        "orchestratorType": "Kubernetes",
+        "kubernetesConfig": {
+        "addons": [
+          {
+            "name": "aad-pod-identity",
+            "enabled": true
+          }
+        ]
+      }
+      },
+      "masterProfile": {
+        "count": 1,
+        "dnsPrefix": "",
+        "vmSize": "Standard_DS2_v2",
+      },
+      "agentPoolProfiles": [
+        {
+          "name": "agentpool",
+          "count": 3,
+          "vmSize": "Standard_DS2_v2",
+          "availabilityProfile": "AvailabilitySet"
```

Comment:
Please add a comment to indicate currently only `"AvailabilitySet"` is supported to avoid users creating using `"VirtualMachineScaleSets"`.